domjudge/organizations.json

```diff
@@ -11,5 +11,6 @@
 {"id": "U-7560", "name": "Université", "formal_name": " Université de Mons ", "country": "NLD"},
 {"id": "U-7642", "name": "Université", "formal_name": " Université Libre de Bruxelles ", "country": "NLD"},
 {"id": "U-2347", "name": "Utrecht", "formal_name": " Utrecht University ", "country": "NLD"},
-{"id": "U-2381", "name": "Vrije", "formal_name": " Vrije Universiteit Amsterdam ", "country": "NLD"}
+{"id": "U-2381", "name": "Vrije", "formal_name": " Vrije Universiteit Amsterdam ", "country": "NLD"},
+{"id": "U-10500", "name": "UM", "formal_name": "Universit\u00e4t Maastricht ", "country": "NLD"}
```

Comment:
@verwoerd I see the German name pop up here again :stuck_out_tongue: Is this auto-imported from the ICPC system (and if so, should it stay like that), or can we manually write the English name?

Reply:
I got this from CMS yes. But I also noticed the short names and formal names of all the other universities are also a bit weird with spaces (and the short names are just wrong)

Reply:
Verwoerd mentioned he was "kicking up the totempole" to get these fixed in the ICPC CMS, we both hope that somebody up there actually cares about this :sweat_smile: 

Still, we can fix this manually in the PCF repo, but perhaps it's nice to remain consistent with the data that Verwoerd submitted. @verwoerd can you share the data and/or update the JSONs in this repo? :slightly_smiling_face:

Reply:
Ah that sounds good. The short names are just an issue on our side though right? Seems it is the first word if the formal_name every time, which is not what is in CMS. For example it has `VU`, not `Vrije`.

Reply:
see #11